src/Parbad.Gateway/PayIr/src/PayIrVerifyAdditionalDataExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Parbad. All rights reserved.
+// Licensed under the GNU GENERAL PUBLIC License, Version 3.0. See License.txt in the project root for license information.
+
+using Parbad.Gateway.PayIr.Internal;
+
+namespace Parbad.Gateway.PayIr
+{
+    public static class PayIrVerifyAdditionalDataExtensions
+    {
```

Comment:
Add a constant here for the additional key.
The key will be used for setting and getting the data.

public const string AdditionalDataKey = "PayIrAdditionalDataKey";